subsys/sd/mmc.c

```diff
@@ -172,8 +184,8 @@ int mmc_card_init(struct sd_card *card)
 
 static int mmc_send_op_cond(struct sd_card *card, int ocr)
 {
-	struct sdhc_command cmd;
-	int ret;
+	struct sdhc_command cmd = {0};
```

Comment:
these changes to the `mmc_send_op_cond` function should be squashed into the first commit that fully implements this function